doc/api/cli.md

```diff
@@ -473,6 +473,8 @@ Node options that are allowed are:
 - `--zero-fill-buffers`
 
 V8 options that are allowed are:
+- `--perf-prof`
+- `--perf-basic-prof`
```

Comment:
Thanks for updating, but please could you maintain the alphabetical sort?

Reply:
I got it.
I sorted options in alphabetical order.